addons/mail/static/src/core/thread_service.js

```diff
@@ -863,7 +863,11 @@ export class ThreadService {
      * @param {Thread} thread
      * @param {string} body
      */
-    async post(thread, body, { attachments = [], isNote = false, parentId, rawMentions }) {
+    async post(
+        thread,
+        body,
+        { attachments = [], isNote = false, parentId, rawMentions = "" } = {}
```

Comment:
`rawMentions` matches the shape of `composer_model:rawMentions`, which is an object containing 2 sets:
```js
rawMentions = {
    partnerIds: new Set(),
    threadIds: new Set(),
};
```
So it's definitely not an empty string